### DISCUSSION TITLE
AutoMerge not happening despite being enabled

### DISCUSSION THREAD

**Discussion by knechtionscoding** [renovatebot/renovate#18248] [Request Help] [answered]
I'm probably misunderstanding how automerge is supposed to be configured.

I've got a package rule for automerge that looks something like:

```
  "packageRules": [
    {
      "matchPaths": [
        "+(k8s/dev/kustomization.yaml)"
      ],
      "groupName": "dev kustomize",
      "stabilityDays": 3,
      "matchManagers": [
        "kustomize"
      ],
      "matchUpdateTypes": [
        "minor",
        "patch",
        "pin",
        "digest"
      ],
      "automerge": true
    },
```

Despite having automerge enabled it I see that PRs submitted by the job have:

`Automerge: Disabled by config.` 

Can you not enable automerge on a specific subset? 

Or is my pattern match incorrect?

**Comment by rarkins** [comment, marked as answer]
I'm not sure matchPaths will work as you expect.

**Reply by knechtionscoding** [reply]
Okay. So how should I use matchPaths then? Or specifically how should I break up packages by path for things like kustomization

**Reply by rarkins** [reply]
Try matchFiles instead

**Reply by knechtionscoding** [reply]
That worked. I suppose if I had done dev/, staging/, prod/ that also would have worked with paths, right?
